### PR TITLE
Changes needed to deploy the mbid-mapping for the new LB startup service

### PR DESCRIPTION
**Pull request by mayhem**
As it says on the tin -- tested and working in production now.

### CODE REVIEW

File: listenbrainz/mbid_mapping/Dockerfile [hunk 1]
```diff
@@ -18,9 +18,9 @@ COPY . /code/mapper
 FROM mbid-mapping-base as mbid-mapping-prod
 
 # service start up scripts
-COPY docker/consul-template.conf /etc/consul-template.conf
 RUN mkdir /etc/service/mapper
```
Comment (amCap1712) [inline listenbrainz/mbid_mapping/Dockerfile:21]: While we are at it, we can probably remove this line. The documentation for [COPY](https://docs.docker.com/engine/reference/builder/#copy) says
> If <dest> doesn’t exist, it is created along with all missing directories in its path.

File: listenbrainz/mbid_mapping/docker/mapper.service [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
```
Comment (amCap1712) [inline listenbrainz/mbid_mapping/docker/mapper.service:1]: The permissions for this file seem to have changed. Just to confirm, that's intended or harmless?

Comment (mayhem) [reply listenbrainz/mbid_mapping/docker/mapper.service:1]: Intended.

**Comment by mayhem** [comment]
I fixed the suggested change -- merging so I can deploy a labs update.

**Pull request merged**